src/pages/images/actions/DeleteImageBtn.tsx

```diff
@@ -33,9 +30,14 @@ const DeleteImageBtn: FC<Props> = ({ image, project }) => {
           void queryClient.invalidateQueries({
             queryKey: [queryKeys.projects, project],
           });
-          notify.success(`Image ${image.properties.description} deleted.`);
+          toastNotify.success(`Image ${image.properties.description} deleted.`);
         },
-        (msg) => notify.failure("Image deletion failed", new Error(msg)),
+        (msg) =>
+          toastNotify.failure(
+            "Error",
+            new Error(msg),
+            `Image ${image.properties.description} deletion failed`,
+          ),
```

Comment:
Why do we sometimes change the title to the message and user "Error" as title?
I think how it was previously was better. The title can already contain the details and we should avoid the generic "Error" title. Wdyt?

Reply:
I added a bit more detail to the error title and I was worried that the title might become too long because we don't really know the length of certain object properties. So I moved the title to the message body instead. But yeah I can change it back to how it was.